### PR TITLE
9 interactive point distance tool

### LINKED ISSUE #9: Interactive Point Distance Tool
Add interaction feature to Projection3D pane so that users can click on points and see calculated distances. 
First cut will use keyboard controls to select either points or groups. (Click vs control click)
Second cut will provide UI overlay options to simplify selection and provide additional Distance metric values.  

- [x] Should provide options to compute point to point and point to group. 
- [x] Point to point will render single line connecting the end points. 
~~- [ ] Point to group will render lines from the first point selected to each point in the grouping.~~
~~- [ ] Investigate Group to Group line distances~~
- [x] Groups will be defined by being part of a label cluster/manifold and distances will be Euclidean.
- [x] Investigate 3D line vs 2D overlay line connections. (might be nice to have both options with 2D lines being dashed and transparent. Oh we could use the 2D version as a cool highlight effect when you hover or select that node or 3D link)
 - [x] Implement additional UI tab in Manifolds pane to manage current distance measurements.

### PR DESCRIPTION
**Pull request by Birdasaur**
This update provides a major new feature for interactively measuring distances between points in the Projections view.
Adds a whole new UI component (as subtab) to Manifold Controller.
Major refactoring of existing UI panes.
Support for reading ZERO pilot data. 

Closes #9

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 The Johns Hopkins University Applied Physics Lab LLC
+   Copyright [yyyy] [name of copyright owner]
```
Comment (samypr100) [inline LICENSE.md:189]: e.g. for example

File: src/main/java/edu/jhuapl/trinity/data/Distance.java [hunk 2]
```diff
@@ -0,0 +1,220 @@
+package edu.jhuapl.trinity.data;
+
+/*-
+ * #%L
+ * trinity-1.0.0-SNAPSHOT
```
[the remark this answers is not in the document]
Comment (Birdasaur) [reply src/main/java/edu/jhuapl/trinity/data/Distance.java:5]: dang I'm not sure what is causing that

File: src/test/java/edu/jhuapl/trinity/javafx/ClipTestApp.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
+package edu.jhuapl.trinity.javafx;
+
+/*-
+ * #%L
+ * trinity-1.0.0-SNAPSHOT
```
Comment (samypr100) [inline src/test/java/edu/jhuapl/trinity/javafx/ClipTestApp.java:5]: another one

File: src/test/java/edu/jhuapl/trinity/javafx/ClipTestMain.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package edu.jhuapl.trinity.javafx;
+
+/*-
+ * #%L
+ * trinity-1.0.0-SNAPSHOT
```
Comment (samypr100) [inline src/test/java/edu/jhuapl/trinity/javafx/ClipTestMain.java:5]: another one

File: LICENSE.md (hunk 1 above)
Comment (Birdasaur) [reply LICENSE.md]: resolved.

File: src/main/java/edu/jhuapl/trinity/data/Distance.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Birdasaur) [reply src/main/java/edu/jhuapl/trinity/data/Distance.java]: resolved. Something about how I was creating new files prior to merging main back into this branch.

File: src/test/java/edu/jhuapl/trinity/javafx/ClipTestApp.java (hunk 3 above)
Comment (Birdasaur) [reply src/test/java/edu/jhuapl/trinity/javafx/ClipTestApp.java:5]: all resolved.

**Pull request merged**